examples/platform/esp32/PigweedLogger.cpp

```diff
@@ -86,6 +86,22 @@ SemaphoreHandle_t * getSemaphore()
     return &esp_log_mutex;
 }
 
+static const char * getLogColorForLevel(esp_log_level_t level)
+{
+    switch (level)
+    {
+    case ESP_LOG_ERROR:
+        return LOG_COLOR_E "E";
+
+    case ESP_LOG_INFO:
+        return LOG_COLOR_I "I";
+
+    default:
+        // default is kept as ESP_LOG_DEBUG
+        return LOG_COLOR_D "D";
+    }
+}
+
 extern "C" void __wrap_esp_log_write(esp_log_level_t level, const char * tag, const char * format, ...)
```

Comment:
One small nit, can we add a comment in `__wrap_esp_log_write` regarding why we are not calling `getLogColorForLevel` like we are in `__wrap_esp_log_writev`. Right now I am not sure why that is done, and future file readers would likely want to know why

Reply:
Good point, I have added the comment explaining the difference difference between two wrappers.